website/documentation/getting-started/common-pitfalls/_index.md

```diff
@@ -0,0 +1,209 @@
+
+Tune `shoot.spec.Kubernetes.kubeReserved` to protect the node (kubelet) in case of a workload pod consuming too much resources. It is very helpful to ensure a high level of stability.
+
+If the usage profile changes over time, the VPA can help a lot to adapt the resource requests / limits automatically.
+
+## Webhooks
+
+### User-Deployed Webhooks in Kubernetes
+
+![](./images/user-webhook.gif)
+
+By default, any request to the API server will go through a chain of checks. Let's take the example of creating a pod. 
+When the resource is submitted to the API server, it will be checked 
+- is the user authorized to perform this action? 
+- is the pod definitionactually valid?
+- are the specified values allowed?
+Additionally, there is the defaulting - like the injection of a the `default` service account's name, if nothing else is specified.
+
+This chain of admission control and mutation can be enhanced by the user. Read about [dynamic admission control for more details](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+`ValidatingWebhookConfiguration`: allow / deny based on custom rules
+
+`MutatingWebhookConfiguration`: change а resource before it is actually stored in etcd (that is, before any other controller acts upon)
+
+`ValidatingWebhookConfiguration` / `MutatingWebhookConfiguration`: 
+- specify for which resources and operations these checks should be executed. 
+- specify how the checks are executed (service)
+- webhook serving pod: perform a review and reply to the `admissionReview` request
+
+![](./images/user-webhook-fail.gif)
+
+What can go wrong (i.e., the webhook is not reachable and thus blocking the creation of the pod or any other operation that the webhook is responsible for)?
+
+- VPN / connection issues can happen both on the side of the seed as well as the shoot
+- The traffic cannot reach the pod (network issue, pod not available)
+- The pod is processing too slow (e.g., because there are too many requests)
+
+### Timeout
+
+![](./images/timeout.png)
+
+Webhooks are a very helpful feature of Kubernetes. However, they can easily be configured to break a shoot cluster. Take the timeout, for example. High timeouts (>15s) can lead to blocking requests of control plane components. That's because most control-plane API calls are made with a client-side timeout of 30s, so if a webhook has `timeoutSeconds=30`, the overall request might still fail as there is overhead in communication with the API server and other potential webhooks.
+
+{{% alert color="info"  title="Recommendation" %}}
+Webhooks (esp. mutating) may be called sequentially and thus adding up their individual timeouts. Even if the `faliurePolicy` is ignored, the timeout will stop the request.
+{{% /alert %}}
+
+### Recommendations
+
+![](./images/webhook-recommendations.png)
```

Comment:
I would suggest to drop the picture and add its content to the written text.

Reply:
The contents of the picture are already included in the written text, from what I can see. I can simply drop the picture and leave the text as is.